### PR TITLE
Make Constant Subsystem Configuration

### PR DESCRIPTION
**Pull request by Noah2357**

### CODE REVIEW

File: robotbase/src/main/java/com/team2357/frc2022/Constants.java [hunk 1]
```diff
@@ -113,13 +109,15 @@
...
+            config.m_isRightInverted = true;
+            config.m_isGyroReversed = true;
+            return config;
+        }
     }
 
-    public final class LIMELIGHT {
+    public static final class LIMELIGHT {
         /** Angle of the Limelight axis from horizontal (degrees) */
         public static final double MOUNTING_ANGLE = 0;
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/com/team2357/frc2022/Constants.java:122]: You need to remove these unused constants, and move their respective comments to the ```GET_LimelightSubsystem_CONFIG```.

File: robotbase/src/main/java/com/team2357/frc2022/Constants.java [hunk 2]
```diff
@@ -131,6 +129,15 @@
 
         /** Target height in inches */
         public static final double VISION_TARGET_HEIGHT = 2;
+
+        public static final LimelightSubsystem.Configuration GET_LimelightSubsystem_CONFIG() {
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/com/team2357/frc2022/Constants.java:133]: ```LIMELIGHT``` should be all caps.

Comment (NolanCampbell2357) [reply robotbase/src/main/java/com/team2357/frc2022/Constants.java:133]: ```SUBSYSTEM`` to, should be ```GET_LIMELIGHT_SUBSYSTEM_CONFIG```

**Review by NolanCampbell2357 — CHANGES_REQUESTED** [review]

File: robotbase/src/main/java/com/team2357/frc2022/Constants.java (hunk 2 above)
Comment (Noah2357) [reply robotbase/src/main/java/com/team2357/frc2022/Constants.java]: fixed

File: robotbase/src/main/java/com/team2357/frc2022/Constants.java (hunk 1 above)
Comment (Noah2357) [reply robotbase/src/main/java/com/team2357/frc2022/Constants.java]: fixed

**Review by NolanCampbell2357 — APPROVED** [review]
Good.

**Pull request merged**